arch/xtensa/src/common/xtensa_asm_utils.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2017, Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
So we are not going to use standard NuttX Apache 2.0 header?

Reply:
We can't use ASF headers here since the code was not donated to the ASF.  It's however in a compatible license so we can use it freely.
I don't know if we can remove the SPDX format though. I guess we can.  At least I can do the formatting needed to pass nxstyle checks.

Reply:
Yeah, let's fix NuttX style issue and maybe we can add Apache-2.0 license text. I think that should be fine

Reply:
I fixed the style, but kept the SPDX license format.

Reply:
I looked into `arch/risc-v/src/mpfs/mpfs_opensbi_utils.S` that has both `SPDX identifier` and licence text so was thinking that we can do the same, but let's keep it as is.